helm/values.yaml

```diff
@@ -16,7 +16,7 @@ container:
   runAsUser: 1000
   readOnlyRootFilesystem: true
   allowPrivilegeEscalation: false
-  userServiceUrl: http://mine-support-user-service.default
```

Comment:
The default userServiceUrl has been reverted to use port 3002. The user service was changed to run on port 80, so the default URL should be  `userServiceUrl: http://mine-support-user-service.default`

Reply:
When/where was the user service updated? It still defaults to 3002 in the [config.js](https://github.com/DEFRA/mine-support-user-service/blob/master/server/config.js)

Reply:
Checked with @johnwatson484, the update was specifically to Helm deployments.

I've reverted back to the correct port.